src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Writing Meaningful Web Content With Semantic HTML
+---
+
+import {
+  SprkDivider,
+} from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Writing Meaningful Web Content with Semantic HTML
+</h1>
+
+Typically, in a word document, writers bold or italicize text to add emphasis to the copy.
+
+When producing web content, **HTML elements** define the structure, purpose and intended meaning of a page's web content.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Main Takeaways
```

Comment:
This is generating an error in [Axe Dev Tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) - Heading levels should only increase by one. This should be an `H2`.